docs/content/configuration/creating_volumereplication.md

```diff
@@ -12,7 +12,7 @@ When replicating a volume, be sure to follow all of the replication configuratio
 
     For more information about PVC configuration, see [Creating a PersistentVolumeClaim (PVC)](creating_pvc.md).
 
-    **Note:** Use the `spec.csi.volumeHandle` of the relevant target PersistentVolume (PV) for the `replicationHandle` value.
+    **Important:** If you are not using policy-based replication, use the `spec.csi.volumeHandle` of the relevant target PersistentVolume (PV) for the `replicationHandle` value. If policy-based replication is being used, **do not** use the `replicationHandle` value. Using the `replicationHandle` while policy-based replication is defined in the VolumeReplicationClass results in the following error message: `got an invalid parameter: replicationHandle`.
```

Comment:
we need to add description about configuring with the VolumeGroup Type for policy-based replication. maybe even new yaml

Reply:
updated FYI @ELENAGER  please review the changes